assets/src/edit-story/components/library/panes/media/mediaUtils.js

```diff
@@ -84,3 +84,29 @@ export const getResourceFromMediaPicker = (mediaPickerEl) => {
     videoId,
   };
 };
+
+/**
+ * Generates a resource object from the upload API response object
+ *
+ * @param {Object} file The uploaded file payload.
+ * @return {Object} Resource object.
+ */
+export const getResourceFromUploadAPI = (file) => {
```

Comment:
This was here before, no? Bad rebase?

Reply:
No. I think you added it in one of the rebases and I just forked it over. It's not on the master right now...